jina/serve/instrumentation/_aio_server.py

```diff
@@ -60,6 +60,8 @@ def telemetry_wrapper(behavior, request_streaming, response_streaming):
         return _wrap_rpc_behavior(next_handler, telemetry_wrapper)
 
     def _intercept_aio_server_unary(self, behavior, handler_call_details):
+        from jina.helper import iscoroutinefunction
```

Comment:
why is this needed?

Reply:
Not needed if we decide to use the `aio.HealthServicer` approach. I was just experimenting if the interceptors can be easily modified to accept any method.